image/cli/mascli/templates/pod-templates/guaranteed/ibm-data-dictionary-assetdatadictionary.yml

```diff
@@ -0,0 +1,56 @@
+# podTemplates
+# =============================================================================
+# This list defines the resources for pod containers/initContainers that get
+# assigned a QoS class of BestEffort. More can be found here:
+# https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod
+podTemplates:
+  - name: graph-store
+    containers:
+      - name: graph-store
+        resources:
+          requests:
```

Comment:
Doesn't it supposed to have the same requests as in operator? https://github.ibm.com/maximoappsuite/ibm-data-dictionary/blob/dev/operator/ibm-data-dictionary/roles/assetdatadictionary/vars/images.yml#L80

Reply:
You are confusing it with `Burstable`. Guaranteed will have requests set to limits. BestEffort will have resources set to empty dict.